_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -272,7 +282,7 @@ Element has ARIA role, but no ARIA states or properties
 
 #### Inapplicable Example 3
 
-`aria-checked` state with empty value
+The `aria-checked` attribute does not have a value.
```

Comment:
Not so much a question about this specific change, but I received a question from my team about the example as a whole. ARIA spec notes for aria-checked:
> undefined (default): The element does not support being checked.

The example to "Accept terms" that doesn't support being checked is strange. Does it not support being checked because it's currently disabled? Adding `aria-disabled="true"` would cause this rule to pass.

Reply:
I'm just going to use a different attribute here I think. You're right, an `undefined` aria-checked prop on a checkbox is odd.